### PR TITLE
Get issue changelog

### PR DESCRIPTION
**Pull request by kobiocdn**
Expose https://developer.atlassian.com/cloud/jira/platform/rest/v2/#api-rest-api-2-issue-issueIdOrKey-changelog-get

### CODE REVIEW

**Comment by MrRefactoring** [comment]
You can also add data to `issue.d.ts`?

File: api/issue.js [hunk 1]
```diff
@@ -779,6 +779,24 @@ function IssueClient(jiraClient) {
...
+    /**
+     * Returns the list of changelogs for the issue with the given key.
+     *
+     * @method getChangelog
+     * @memberOf IssueClient#
+     * @param {Object} opts The options to pass to the API.  Note that this object must contain EITHER an issueId or
+     *     issueKey property; issueId will be used over issueKey if both are present.
+     * @param {string} [opts.issueId] The id of the issue.  EX: 10002
+     * @param {string} [opts.issueKey] The Key of the issue.  EX: JWR-3
+     * @param [callback] Called after the changelog is retrieved.
```
Comment (MrRefactoring) [inline api/issue.js]: Lacks `QUERY PARAMETERS` section

Comment (kobiocdn) [reply api/issue.js]: Good catch.  Added opts.qs support for pagination.

Comment (MrRefactoring) [reply api/issue.js]: I think it's better to bring startAt and maxResults to the top level (From opts.qs to opts)

Comment (kobiocdn) [reply api/issue.js]: Moved startAt and maxResults to top level.  Tested against one of my issues with 57 changelog entries by lowering maxResults and confirming pagination returns expected results.

**Comment by MrRefactoring** [comment]
@kobiocdn you also need to add a reference to this method in `index.d.ts` and `README.md`

**Comment by MrRefactoring** [comment]
Thank you for the contribution!

**Pull request merged**